wagtail_ab_testing/views.py

```diff
@@ -226,14 +237,28 @@ def __init__(self, request, **kwargs):
             self.default_item = None
 
     def render_html(self):
-        return render_to_string(self.template, {
-            'default_menu_item': self.default_item.render_html(self.request, self.context),
-            'show_menu': bool(self.menu_items),
-            'rendered_menu_items': [
-                menu_item.render_html(self.request, self.context)
-                for menu_item in self.menu_items
-            ],
-        }, request=self.request)
+        if WAGTAIL_VERSION >= (3, 0):
```

Comment:
Thought: Why not just pin this to >= 2.15 then?

Reply:
Changed